src/main/scala/org/globalforestwatch/summarystats/gfwpro_dashboard/GfwProDashboardData.scala

```diff
@@ -15,10 +15,25 @@ case class GfwProDashboardData(
 
   /** Location intersects Integrated Alert tiles, integrated alerts are possible */
   glad_alerts_coverage: Boolean,
+  /** Total hectares in location geometry */
+  total_ha: ForestChangeDiagnosticDataDouble,
   /** How many hectares of location geometry had tree cover extent > 30%  in 2000 */
   tree_cover_extent_total: ForestChangeDiagnosticDataDouble,
-  /** Integrated alert count within location geometry grouped by day */
+  /** Integrated alert count within location geometry grouped by day, over the whole
+    * area and including all confidences */
   glad_alerts_daily: GfwProDashboardDataDateCount,
+  /** Integrated alert count within location geometry grouped by day, and further
+    * subdivided by cross-product of cover area (all, SBTN area only, JRC area only)
+    * x alert confidence (lo, med, hi) */
+  glad_alerts_daily_all_lo: GfwProDashboardDataDateCount,
```

Comment:
Since you're adding tons of new fields with "glad" in the name, but we technically switched to integrated alerts, should we use this opportunity to switch the name to use "integrated_alerts" or generically "deforestation_alerts"? At least for the new fields, this way Pro won't need to migrate to a new schema later

Reply:
OK, good idea.  I'll check with Kinshuk if it is ok to change the names for the new columns (I'm assuming it will be fine).  Also, I'll see if we can actually switch the names of the existing columns this time around - but the requirement is that we synchronize the updates to geotrellis and the GFWPro App code, so that is trickier to set up.

Thanks for the review!

Reply:
OK, I did the rename of the new columns.  I also duplicated the glad_alerts_coverage column to integrated_alerts_coverage, as suggested by Kinshuk.  The plan is that GFWPro will eventually only need the daily alerts (as broken down into all the sub-categories in this change).  So, GFWPro should eventually be able to transition to depending only on the integrated_alerts_* columns.  Once it has fully transitioned, we should be able to asynchronously remove all the glad_alerts_* columns.  Thanks for the comment!